mrtt-ui/src/components/ProjectDetailsForm.js

```diff
@@ -67,21 +69,22 @@ const ProjectDetailsForm = () => {
     }
     // make axios PUT request
     axios
-      .put(url, preppedData)
+      .patch(url, preppedData)
```

Comment:
See[ this thread ](https://sparkgeo.slack.com/archives/C036D4TLVPT/p1652308605540279)